### ISSUE TITLE
Final version of fetchAstronauts() function not working.

### ISSUE THREAD

**Issue by Richard-Burd** [learn-co-curriculum/redux-thunk-readme#16]
The `fetchAstronauts()` function does not dispatch JSON data to the reducer.  Although this lesson does not actually contain a functioning react-redux application, it describes an astronauts container with a commensurate action and reducer.  I built out the astronaut fetch & display functionality (exactly as described in this lesson) in a sandbox repo over [here](https://github.com/Richard-Burd/react-redux-sandbox/tree/master/React-Redux-Tutorials/react-redux-demo) and corrected the `fetchAstronauts()` action [here](https://github.com/Richard-Burd/react-redux-sandbox/blob/master/React-Redux-Tutorials/react-redux-demo/src/redux/astronaut/astronautActions.js) to get this code working properly.

In short, line 354 of the README.md currently reads:

`.then(astronauts => dispatch({ type: 'ADD_ASTRONAUTS', astronauts }));`

...but it needs to be corrected to something like:

`.then(responseJSON => {
      dispatch({ type: 'ADD_ASTRONAUTS', astronauts: responseJSON.people })`

...in order for it to work.
